src/org/usfirst/frc/team2335/robot/commands/ShootGroup.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class ShootGroup extends CommandGroup {
+    
+    public ShootGroup()
+    {
+    	DriverStation.reportWarning("Running", true);
```

Comment:
This is different from the smart dashboard, this is a different way of printing text, I'll go over it at the meeting if you'd like.